.github/workflows/daily.yaml

```diff
@@ -0,0 +1,40 @@
+name: LuxOS-Tooling Daily
+
+on:
+  # schedule:
+  #   - cron: "0 0 * * *"
```

Comment:
i think. this is needed

Reply:
added the token
shared a successful run in the description
added the schedule back